iOS_SDK/OneSignalSDK/OneSignalCore/Source/OneSignalCommonDefines.h

```diff
@@ -33,8 +33,8 @@
 // Networking
 #define OS_API_VERSION @"1"
 #define OS_API_ACCEPT_HEADER @"application/vnd.onesignal.v" OS_API_VERSION @"+json"
-#define OS_API_SERVER_URL @"https://api.staging.onesignal.com/"
-#define OS_IAM_WEBVIEW_BASE_URL @"https://staging.onesignal.com/"
```

Comment:
The bundle id and group have staging in them but the app points at prod, just a heads up.

Reply:
Ah I see, the app points at prod for sending requests, but the bundle ID is different. That's the bundle ID for this app_id.

Reply:
Ohh ahh wow, good call. I wasn't thinking about that this app_id only exists in the staging environment. This is why I was having some issues.